.github/dockerfiles/ov_build/ubuntu_20_04_x64/Dockerfile

```diff
@@ -0,0 +1,73 @@
+FROM openvinogithubactions.azurecr.io/dockerhub/ubuntu:20.04
+
+USER root
+
+# APT configuration
+RUN echo 'Acquire::Retries "10";' > /etc/apt/apt.conf && \
+    echo 'APT::Get::Assume-Yes "true";' >> /etc/apt/apt.conf && \
+    echo 'APT::Get::Fix-Broken "true";' >> /etc/apt/apt.conf && \
+    echo 'APT::Get::no-install-recommends "true";' >> /etc/apt/apt.conf
+
+ENV DEBIAN_FRONTEND="noninteractive" \
+    TZ="Europe/London"
+
+RUN apt-get update && \
+    apt-get install software-properties-common && \
+    add-apt-repository --yes --no-update ppa:git-core/ppa && \
+    add-apt-repository --yes --no-update ppa:deadsnakes/ppa && \
+    apt-get update && \
+    apt-get install \
+        curl \
+        git \
+        ca-certificates \
+        gpg-agent \
+        tzdata \
+        libtbb2 \
```

Comment:
I suppose we don't need to install TBB, because `install_build_dependencies.sh` does it

Furthermore, in recent PRs we fully switched to custom TBB shipped with OpenVINO https://github.com/openvinotoolkit/openvino/pull/24359 and don't required TBB on the system at all.

Reply:
Removed.